lib/loader/index.d.ts

```diff
@@ -99,19 +102,19 @@ interface ASUtil {
 /** Asynchronously instantiates an AssemblyScript module from anything that can be instantiated. */
 export declare function instantiate<T extends {}>(
   source: WebAssembly.Module | BufferSource | Response | PromiseLike<WebAssembly.Module | BufferSource | Response>,
-  imports?: ImportsObject
+  imports?: Imports
 ): Promise<ResultObject & { exports: ASUtil & T }>;
 
 /** Synchronously instantiates an AssemblyScript module from a WebAssembly.Module or binary buffer. */
 export declare function instantiateSync<T extends {}>(
   source: WebAssembly.Module | BufferSource,
-  imports?: ImportsObject
+  imports?: Imports
 ): ResultObject & { exports: ASUtil & T };
```

Comment:
Not directly related to this PR, but according to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/instantiate) there's also a slight variance here: If a `WebAssembly.Module` is provided, this API returns just a `WebAssembly.Instance`, otherwise a `ResultObject`.

Reply:
not sure about this. but if you see this is reasonable we could do thus in separate pr